libs/langchain-weaviate/tsconfig.json

```diff
@@ -3,11 +3,13 @@
   "compilerOptions": {
     "outDir": "../dist",
     "rootDir": "./src",
-    "target": "ES2021",
-    "lib": ["ES2021", "ES2022.Object", "DOM"],
-    "module": "ES2020",
-    "moduleResolution": "nodenext",
+    "target": "ESNext",
+    // "lib": ["ES2021", "ES2022.Object", "DOM"],
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```

Comment:
Unfortunately with our current build process it's important to keep these as they were. If you were seeing errors here it'd likely be due to VSCode using its internal TS version rather than the workspace version. You can hit `ctrl-shift-p` (or `cmd-shift-p` on mac) and type something like "TypeScript version" into the command palette and it'll let you select the workspace version instead.

Reply:
With these changes I expect the CJS build will be broken.